dynawo/sources/Models/CPP/ModelNetwork/DYNModelRatioTapChanger.h

```diff
@@ -20,6 +20,7 @@
 #ifndef MODELS_CPP_MODELNETWORK_DYNMODELRATIOTAPCHANGER_H_
 #define MODELS_CPP_MODELNETWORK_DYNMODELRATIOTAPCHANGER_H_
 
+#include "DYNModelConstants.h"
```

Comment:
Same comment as above

Reply:
It is necessary here since we mention DYN::VALDEF